lib/daysOfTheWeek.js

```diff
@@ -9,3 +9,11 @@ module.exports.getDayName = (date) => {
 
   return ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'][passedDate.getDay()];
 };
+
+module.exports.next = (dow) => {
+  const dows = ['Sunday','Monday','Tuesday',  'Wednesday','Thursday', "Friday",'Saturday'];
+  let days = (dows.indexOf(dow)-(new Date()).getDay());
```

Comment:
This `days` variable should probably be renamed to something more meaningful.  It's also hard to tell what this line of code is doing.  It might be a good idea to put a comment here so someone in the future will know what's happening.